website/content/docs/platform/k8s/helm/examples/standalone-tls.mdx

```diff
@@ -23,30 +23,37 @@ There are three variables that will be used in this example.
 ```bash
 # SERVICE is the name of the Vault service in Kubernetes.
 # It does not have to match the actual running service, though it may help for consistency.
-SERVICE=vault-server-tls
+export SERVICE=vault-server-tls
 
 # NAMESPACE where the Vault service is running.
-NAMESPACE=vault-namespace
+export NAMESPACE=vault-namespace
 
 # SECRET_NAME to create in the Kubernetes secrets store.
-SECRET_NAME=vault-server-tls
+export SECRET_NAME=vault-server-tls
 
 # TMPDIR is a temporary working directory.
-TMPDIR=/tmp
+export TMPDIR=/tmp
+
+# CSR_NAME will be the name of our certificate signing request as seen by Kubernetes.
+export CSR_NAME=vault-csr
```

Comment:
May want to update the opening sentence here since it states `There are three variables...`

Reply:
:+1: